traffic_ops/app/db/migrations/20200402133144_create_topology_tables.sql

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
@zrhoffman - the date on this migration is bad now fyi. keep an eye on it.

Reply:
yeah I DMed him after I merged Matt's LE PR yesterday. I wish we had like a git commit hook or something that would prevent merging a PR with a DB migration that's earlier than one that already exists.

Reply:
yep, these migration dates have always been a problem :(

Reply:
Updated the timestamp in d7e2b857c5.